velox/docs/develop/checklist-functions.rst

```diff
@@ -0,0 +1,75 @@
+=================================================
+Check the items when you add or modify a function
+=================================================
+
+Pull request title
+------------------
+
+* The title should be ``Add Spark[Presto] function xxx``.
+
+Pull requeste description
+-------------------------
+The PR description will be your commit message, please pay attention to it.
+
+* Add the link of Spark implementation for Spark function.
```

Comment:
nit: how about `Add the link of the function implementation from a specific engine, i.e., Spark and Presto.`